app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -24,8 +35,26 @@ const DelegationValidation = ({delegationValidationError, txSuccessTab}) =>
     )
   )
 
-const StakePoolInfo = ({pool, gettingPoolInfo}) => {
-  const {fixedCost, homepage, margin, ticker, name, validationError} = pool
+type StakePoolLabelProps = {
+  isTicker: boolean
+  isPoolHash: boolean
+}
+
+const StakePoolLabel = ({isTicker, isPoolHash}: StakePoolLabelProps): h.JSX.Element => (
+  <Fragment>
+    <span className={isTicker ? 'valid' : ''}>Ticker</span> or{' '}
```

Comment:
Is `valid` a good name? I think something like `highlight' would be bit better. Css doesnt need to know about validity.

Reply:
Refactored